sdcm/nemesis.py

```diff
@@ -1716,6 +1712,33 @@ def disrupt_network_start_stop_interface(self):  # pylint: disable=invalid-name
         finally:
             self.target_node.remoter.run("sudo /sbin/ifup eth1")
 
+    def disrupt_add_del_nodes(self):
+        add_nodes_number = self.tester.params.get('add_node_cnt', default=1)
+        init_nodes_number = self.tester.params.get("n_db_nodes")
+
+        max_nodes_number = add_nodes_number + init_nodes_number
```

Comment:
change the order for readability: init_nodes_number + add_nodes_number

Reply:
fixed